package.json

```diff
@@ -5,22 +5,27 @@
     "dev": "nuxt dev",
     "generate": "nuxt generate",
     "preview": "nuxt preview",
+    "codegen": "graphql-codegen",
```

Comment:
create one more file `codegen.yml` at the root of the app and add the following content

```
schema: https://rickandmortyapi.com/graphql
documents: './composables/*.ts'
generates:
  ./types/graphql/generated.ts:
    plugins:
      - typescript
      - typescript-operations
      - typescript-vue-apollo-smart-ops
```